src/sentry/static/sentry/app/components/events/contexts/default.jsx

```diff
@@ -10,7 +10,17 @@ class DefaultContextType extends React.Component {
   };
 
   render() {
-    return <ContextBlock data={this.props.data} alias={this.props.alias} />;
+    const knownData = Object.entries(this.props.data)
```

Comment:
I would create a method named for example `getKnownData` and place it outside of the render

Reply:
done